packages/calling/src/CallingClient/calling/call.test.ts

```diff
@@ -165,6 +165,47 @@ describe('Call Tests', () => {
     expect(call).toBeTruthy();
   });
 
+  it('should log a warning when sending a digit fails', async () => {
+    const tone = '1';
+    const errorMessage = 'Failed to send digit';
+
+    // Mock the mediaConnection object
+    const mockMediaConnection = {
+      insertDTMF: jest.fn(() => {
+        throw new Error(errorMessage);
+      }),
+    };
+
+    const callManager = getCallManager(webex, defaultServiceIndicator);
+
+    const call = callManager.createCall(dest, CallDirection.OUTBOUND, deviceId, mockLineId);
+
+    const realMediaConnection = call['mediaConnection'];
+    // Set the mock mediaConnection object
+    call['mediaConnection'] = mockMediaConnection;
+
+    // Spy on the log.warn method
+    const logWarnSpy = jest.spyOn(log, 'warn');
+
+    // Call the sendDigit method
+    call.sendDigit(tone);
+
+    // Expect the log.warn method to be called with the error message
+    expect(logWarnSpy).toHaveBeenLastCalledWith(
+      'Unable to send digit on call: Failed to send digit',
```

Comment:
Should we use `` `Unable to send digit on call: ${errorMessage}` `` ?

Reply:
Yes this makes sense, I've made the change